core/Changes/Model.php

```diff
@@ -58,7 +58,8 @@ public function addChanges(string $pluginName): void
             $pluginManager &&
             $pluginManager->isValidPluginName($pluginName) &&
             $pluginManager->isPluginInFilesystem($pluginName) &&
-            $pluginManager->isPluginActivated($pluginName))
+            $pluginManager->isPluginActivated($pluginName)
+        )
         {
```

Comment:
Note: Those will be moved to one line when activating the rule `Squiz.ControlStructures.ControlSignature.SpaceAfterCloseParenthesis`